app/assets/javascripts/components/core.js.coffee

```diff
@@ -16,6 +16,12 @@ $ ->
   # Select2 Selects
   $(".select2").select2(width: 'resolve')
 
+  $(".select2-linked-tags").select2(
+    width: 'resolve',
+    formatSelection: (obj) ->
+      "<a href=\"#{this.element.data('urlPrefix')}/#{obj.id}/edit\" onClick=\"Utils.select2TagClickHandler(event, this)\">#{obj.text}</a>"
```

Comment:
Include underscore.js and use `_.escape obj.text` to be sure we don't have a HTML injection?